ui/pages/create-account/institutional/connect-custody/account-list.js

```diff
@@ -0,0 +1,222 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../../../../components/ui/button';
+import CustodyLabels from '../../../../components/institutional/custody-labels';
+import { SWAPS_CHAINID_DEFAULT_BLOCK_EXPLORER_URL_MAP } from '../../../../../shared/constants/swaps';
+import { CHAIN_IDS } from '../../../../../shared/constants/network';
+import { shortenAddress } from '../../../../helpers/utils/util';
+import Tooltip from '../../../../components/ui/tooltip';
+import {
+  Color,
+  TextVariant,
+  JustifyContent,
+  BLOCK_SIZES,
+  DISPLAY,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import Box from '../../../../components/ui/box';
+import {
+  Text,
+  Label,
+  Icon,
+  ICON_NAMES,
+  ICON_SIZES,
+  ButtonLink,
+} from '../../../../components/component-library';
+import { useCopyToClipboard } from '../../../../hooks/useCopyToClipboard';
+
+const getButtonLinkHref = (account) => {
+  const url = SWAPS_CHAINID_DEFAULT_BLOCK_EXPLORER_URL_MAP[CHAIN_IDS.MAINNET];
+  return `${url}address/${account.address}`;
+};
+
+export default function CustodyAccountList({
+  rawList,
+  accounts,
+  onAccountChange,
+  selectedAccounts,
+  onCancel,
+  onAddAccounts,
+  custody,
+}) {
+  const t = useI18nContext();
+  const [copied, handleCopy] = useCopyToClipboard();
+  const tooltipText = copied ? t('copiedExclamation') : t('copyToClipboard');
+  const disabled = Object.keys(selectedAccounts).length === 0;
+  const buttonProps = {};
+  if (disabled) {
+    buttonProps.disabled = true;
+  }
+
+  return (
+    <>
+      <Box paddingTop={4} paddingRight={7} paddingBottom={7} paddingLeft={7}>
+        <Box
+          display={DISPLAY.FLEX}
+          flexDirection={['column']}
+          width={BLOCK_SIZES.FULL}
+          className="custody-account-list"
+          data-testid="custody-account-list"
+        >
+          {accounts.map((account, idx) => (
+            <Box
+              display={DISPLAY.FLEX}
+              className="custody-account-list__item"
+              key={account.address}
+            >
+              <Box
```

Comment:
Question: do we still need this DOM if `rawList` exists as a placeholder?

Reply:
I'm not sure, wdyt @zone-live ?